### DISCUSSION TITLE
Color scheme during build

### DISCUSSION THREAD

**Discussion by cociweb** [arendst/Tasmota#11989] [Q&A] [answered]
Hello,
I'm trying to create a custom build.
It would make me easier, if I can identify the build for the First look. So I decided to go on with custom color scheme. It's ok, if I define it with the webcolor parameter, but it would be nice if I would be able to do it during compile.

So I've put the followings into the `user_config_override.h` file, but nothing happens:

`

      #define COLOR_TEXT                  "#cccccc"    // [WebColor1] Global text color
      #define COLOR_BACKGROUND            "#2f3133"    // [WebColor2] Global background color 
      #define COLOR_FORM                  "#3d3f41"    // [WebColor3] Form background color
      #define COLOR_INPUT_TEXT            "#dddddd"    // [WebColor4] Input text color
      #define COLOR_INPUT                 "#293134"    // [WebColor5] Input background color
      #define COLOR_CONSOLE_TEXT          "#ffb000"    // [WebColor6] Console text color
      #define COLOR_CONSOLE               "#293134"    // [WebColor7] Console background color
      #define COLOR_TEXT_WARNING          "#ff5661"    // [WebColor8] Warning text color
      #define COLOR_TEXT_SUCCESS          "#008000"    // [WebColor9] Success text color
      #define COLOR_BUTTON_TEXT           "#ffffff"    // [WebColor10] Button text color
      #define COLOR_BUTTON                "#ec7600"    // [WebColor11] Button color
      #define COLOR_BUTTON_HOVER          "#bf5f00"    // [WebColor12] Button color when hovered over
      #define COLOR_BUTTON_RESET          "#d43535"    // [WebColor13] Restart/Reset/Delete button color
      #define COLOR_BUTTON_RESET_HOVER    "#931f1f"    // [WebColor14] Restart/Reset/Delete button color when hovered over 
      #define COLOR_BUTTON_SAVE           "#47c266"    // [WebColor15] Save button color
      #define COLOR_BUTTON_SAVE_HOVER     "#5aaf6f"    // [WebColor16] Save button color when hovered over
      #define COLOR_TIMER_TAB_TEXT        "#ffffff"    // [WebColor17] Config timer tab text color
      #define COLOR_TIMER_TAB_BACKGROUND  "#999999"    // [WebColor18] Config timer tab background color
      #define COLOR_TITLE_TEXT            "#bc4d90"    // [WebColor19] Title text color
`

These are the colors of the Helloween theme.

What am I doing wrong?

Thank you for your hints!

Best regards,
Cociweb

**Comment by sfromis** [comment, marked as answer]
When editing `user_config_override.h` it happens that people put changes into the comment block instead of right at the end. Did that also happen to you?

**Reply by cociweb** [reply]
No, I'm sure it's not the case in my point of view (the other settings are compiled well)

**Reply by ascillato** [reply]
If you already have Tasmota on that device, you need to do `reset 5` in order to set your settings to the default that you have defined in user_config_override.h

**Reply by cociweb** [reply]
Oh! Great! That was in my case!

Thank you guys!
